components/ReleaseSort/ReleaseSort.jsx

```diff
@@ -75,11 +131,17 @@ export default function ReleaseSort({ releases, onChange }) {
         value={selected}
         onChange={handleChange}
       >
-        {sortOptions.map(({ label, value }) => (
-          <option value={value} key={value}>
-            {label}
-          </option>
-        ))}
+        {isDashboard
+          ? sortDashboardOptions.map(({ label, value }) => (
+              <option value={value} key={value}>
+                {label}
+              </option>
+            ))
+          : sortProfileOptions.map(({ label, value }) => (
+              <option value={value} key={value}>
+                {label}
+              </option>
+            ))}
```

Comment:
I think we can do this better, but we can tackle in a future refactor. We should pass in the sort options we want from the parent. For example, in the dashboard page code you have `sortOptions` that get passed into the component: `<ReleaseSort options={sortOptions}>`. Profile page would do that same with it's own options.

While what you have works, it's not scalable. Imagine we want the sort component to have different options for another page. Then you'd need to pass through _another_ boolean prop, which gets pretty messy.

However, for now, try something like this with the above changes:

```jsx
const options = isDashboard ? [...dashboardOptions, ...sharedOptions] : sharedOptions;

// ...

{options.map(({ label, value }) => (
  <option value={value} key={value}>
    {label}
  </option>
)) : null}
```

Reply:
You could also conditionally spread dashboard options within the same array, but it might be a little less readable:

```js
const options = [...(isDashboard ? dashboardOptions : []), ...sharedOptions];
```